framework/include/userobject/InterfaceQpMaterialPropertyBaseUserObject.h

```diff
@@ -0,0 +1,120 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "InterfaceQpUserObjectBase.h"
+
+// Forward declarations
+template <typename T = Real>
+class InterfaceQpMaterialPropertyBaseUserObject;
+
+// template <>
+// InputParameters validParams<InterfaceQpMaterialPropertyBaseUserObject<>>();
```

Comment:
Remove this entirely

Reply:
ups I'm sorry about that
I tried so many different ways that  I confused myself.

I have one question though: all the interfaceUO use neighbors and should have relationship manger. What is need is a relationship manager with a layer of connected nodes, not element (think at tets or tri meshes ). This will ensure that when coupling variables we don't miss stuff. is there any example for this?